tema21ian/src/ExerciseOne/MaximTest.java

```diff
@@ -0,0 +1,44 @@
+package ExerciseOne;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+public class MaximTest {
+
+    private Maxim maxim;
+    private int expected;
+    private int firstParameter;
+    private int secondParameter;
+
+    public MaximTest(int expected, int firstParameter, int secondParameter) {
+        this.expected = expected;
+        this.firstParameter = firstParameter;
+        this.secondParameter = secondParameter;
+    }
+
+    @Before
+    public void setup() {
+        maxim = new Maxim();
+        System.out.println(" test ... ");
+    }
+@Parameterized.Parameters
+    public static Collection<Object[]> testConditions() {
+        return Arrays.asList(new Object[][]{
+                {3, 11, 8},
```

Comment:
from a test point of view this should work. If you use parameters, why don't you test with multiple values.
Also, this first test case should fail, you expect the max between 11 and 8 to be 3?